packages/inference/package.json

```diff
@@ -55,8 +55,9 @@
 		"export-templates": "tsx scripts/export-templates.ts"
 	},
 	"dependencies": {
+		"@huggingface/jinja": "workspace:^",
 		"@huggingface/tasks": "workspace:^",
-		"@huggingface/jinja": "workspace:^"
+		"@modelcontextprotocol/sdk": "^1.9.0"
```

Comment:
If it's not too troublesome, it would be nice to either reimplement what's needed or vendor part of the code eg as we did with `fetch-event-source`: https://github.com/huggingface/huggingface.js/blob/b033ad193002c32091e196bf4936b28450eaae0a/packages/inference/src/vendor/fetch-event-source/parse.ts

See https://www.npmjs.com/package/@modelcontextprotocol/sdk?activeTab=dependencies this adds 10 sub-dependencies including `express`

Reply:
sorry i should have mentioned, i'll publish this as another sub-package @coyotte508